keyboards/vortex/m0110a/info.json

```diff
@@ -0,0 +1,853 @@
+{
+    "manufacturer": "Vortex",
+    "keyboard_name": "M0110A",
+    "maintainer": "Gsky",
+    "bootloader": "wb32-dfu",
+    "bootmagic": {
+        "matrix": [1, 6]
+    },
+    "diode_direction": "COL2ROW",
+    "dynamic_keymap": {
+        "layer_count": 5
+    },
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "indicators": {
+        "num_lock": "C14",
+        "caps_lock": "C15"
+    },
+    "matrix_pins": {
+        "cols": ["A0", "A1", "A2", "A3", "A4", "A8", "A9", "A10"],
+        "rows": ["B0", "B1", "B2", "B3", "B4", "B5", "B6", "B7", "B8", "B9", "B10", "B11", "B12", "B13"]
+    },
+    "processor": "WB32FQ95",
+    "qmk": {
+        "tap_keycode_delay": 10,
+        "locking": {
+            "enabled": true,
+            "resync": true
+        }
+    },
+    "url": "",
+    "usb": {
+        "device_version": "0.0.2",
+        "pid": "0xE39C",
+        "vid": "0x342D"
+    },
+    "eeprom": {
+        "driver": "wear_leveling",
+        "wear_leveling": {
+            "driver": "spi_flash",
+            "backing_size": 8192
+        }
+    },
+    "layouts": {
```

Comment:
Trying to make sense of this. 

- `[10, 2]` only gets used on 1800 layout boards in ANSI layout, as the 1.5u Backslash key
- `[8 ,4]` only gets used as the 1.25u RWin key, and is only available in the 60% + Numpad layout, and not the mini 1800 layout with arrow keys
- 2u Backspace is `[10, 1]`, and Split Backspace is `[10, 1]` and `[6, 5]`

Do I have all of that correct?